site/kubernetes/kube-addon.md

```diff
@@ -270,6 +271,15 @@ TCP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 ```
 
+### <a name="key-points"></a> Things to watch out for
+
+- Don't turn on `--masquerade-all` on kube-proxy: this will change the
+  source address of every pod-to-pod conversation which will make it
+  impossible to correctly enforce network policies that restrict which
+  pods can talk.
+- If you do set the `--cluster-cidr` option on kube-proxy, make sure
+  it matches the `IPALLOC_RANGE` given to Weave Net (see below)
+
 ## <a name="configuration-options"></a> Changing Configuration Options
 
```

Comment:
Don't use `--masquerade-all` on kube-proxy: this setting changes the
source address of every pod-to-pod conversation which makes it
impossible to correctly enforce the network policies that restrict which
pods can communicate.

If you set the `--cluster-cidr` option on kube-proxy, ensure
it matches the `IPALLOC_RANGE` given to Weave Net (see below)

Reply:
I don't think "the network policies" gives the right meaning - there are no policies by default; only ones you write, and you have to write ones which seek to restrict pods before you hit the problem (you can still write ones that restrict ports, for instance)